nav2_bringup/bringup/params/nav2_params_namespaced.yaml

```diff
@@ -0,0 +1,164 @@
+navigation_namespace:  # this is substituted with the actual robot name during launch
```

Comment:
I'm curious, can you load a .yaml into a namespace? That way you dont have to make identical yaml files that are only changed by the namespace the rest of the file is within?

Reply:
Will check if there's such feature in ROS.

Yaml doesn't support including another yaml :disappointed:

Reply:
@SteveMacenski I don't think this is currently supported by the [parser](https://github.com/ros2/rcl/blob/3e0efcef34b8da59bee0e5e0bcdb19577e6173cf/rcl_yaml_param_parser/include/rcl_yaml_param_parser/parser.h#L53)

Reply:
An alternative is to have a launch substitution which creates a namespaced YAML from the non-namespaced one. I'll try that.

Reply:
Pushed a commit that puts the yaml file on a namespace and got rid of the duplicated yaml file.

Reply:
@orduno dude, go you, that's awesome. 

That's also something that really should be up stream but that's not important for this PR, just adding to my unending list of ecosystem tickets to fix